packages/nc-gui/composables/useAttachment.ts

```diff
@@ -0,0 +1,35 @@
+import fileNotFoundImgSrc from '~/assets/img/file-not-found.png'
+
```

Comment:
To avoid breaking the asset path in the production build we can do it at the template level by creating a new component for rendering images.

Something like the following : 


In the main component use it like this: 

```vue

<LazyAttachmentImage :srcs="srcs" />

```
The `srcs` is an array of URLs populated using `item.url` and `item.path`

AttachmentImage component:

```vue
<script setup lang="ts">
  const props = defineProps<{srcs: string[]}>

  const index = ref(0)

  const onError () => index.value++
</script>
<template>
   <img v-if="index < props.srcs.length" :src="props.srcs[index]" @onerror="onError"/>
   <img v-else src="~/assets/img/file-not-found.png" />
</template>
```